docs/run/hardware.mdx

```diff
@@ -1,101 +1,100 @@
 
-        ```python
-        Gate(gate='cx', name='cx4_3', parameters=[Nduv('2020-06-06T10:03:18Z', gate_error, , 0.009793933588297388), Nduv('2020-06-06T10:03:18Z', gate_length, ns, 362.66666666666663)], qubits=[4, 3])
-        ```
+    > *   <span id="coupling-map">**Topology diagram** or **coupling map** </span> - A diagram that indicates the pairs of qubits that support two-qubit gate operations between them. This is also called the coupling map or connectivity. Qubits are represented as circles and the supported two-qubit gate operations are displayed as lines connecting the qubits.
+    > *   **Individual qubit properties** - Shows the selected property for each qubit on the system. You can view the frequency, T1, T2, Anharmonicity, probability measurements, error rates, and so on.
 
+*   **Your access instance** - Instances that you can use. Click the arrow on the right to expand or collapse this section. For each instance, you can see the following information:
 
-## System versioning
+    *   **Max shots** - The maximum number of times you can execute a single circuit on a system. The number of shots taken determines the precision of the output probability distribution over repeated executions.
+    *   **Max circuits** - The maximum number of quantum circuits that you can submit to this system at one time.
+    *   **Max qubits per pulse gate** - The maximum number of qubit arguments allowed to a gate.
+    *   **Max channels per pulse gate** - The maximum number of channels you can refer to within a pulse schedule. Typically each qubit is associated with a drive channel, a measure channel, an acquisition channel, and then auxiliary control channels for things like cross resonance.
+    *   **Usage** - Click the link to see the jobs that you have run on this system.
 
-Each system has a version number in the form X.Y.Z (major.minor.revision). A circuit compiled for a given system version number is guaranteed to run on that system. If the revision number changes, the circuit will continue to run. If the major or minor number changes, the circuit is not guaranteed to run, although it may do so. The conditions under which a version number may change are listed below:
+## View system configuration
 
-### Major version
+View system configuration values by selecting a system on the [Compute resources page.](https://quantum-computing.ibm.com/services/resources?services=systems) The three tabs in the Calibration data section let you choose how to view the calibration data; the Map view tab is automatically selected.
 
-The major version will increment for system changes such as:
+![../../../\_images/exp-card.png](/images/migration/exp-card.png)
```

Comment:
What's up with the `[]` portion? That is usually supposed to be a human readable description

Reply:
no idea! This was pre-existing... what should it be?

Reply:
A concise human readable description of the image, like alt text. "ibm_quito system page" for example

Reply:
Ah I understand -- I was misreading the code and hadn't realized this is what was rendering in the html 😆 ok let me look at that